build/helm-package.sh

```diff
@@ -30,5 +30,7 @@ if [ ! -f "$FILE" ]; then
 fi
 
 sudo mv linux-amd64/helm /usr/local/bin/helm
-RESULT_STR=$(helm package ./k8s/helm-chart/dynatrace-gcp-function -d ./artefacts)
-mv ./${RESULT_STR##* } ./artefacts/dynatrace-gcp-function.tgz
\ No newline at end of file
+mkdir ./helm-deployment-package
```

Comment:
Since we are not using the **helm** anymore there's no need to create wget it and check if it's available.